Mathlib/Order/README.md

```diff
@@ -0,0 +1,22 @@
+# Order
+
+This folder contains order theory in a broad sense.
+
+## Order hierarchy
+
+The basic order hierarchy is split across a series of subfolders that each depend on the previous:
+* `Order.Preorder` for preorders, partial orders
+* `Order.Lattice` for lattices, linear orders
```

Comment:
Having a subfolder split between partial orders and linear orders feels strange to me, particularly since if I'm looking for a result in a particular linear order, I shouldn't need to think about if it also holds in partial orders. So I'd suggest linear orders should go in the same subfolder as partial orders. Also, lattices which aren't linear are also useful things to study, so I'm not sure that lattice and linear orders belong together either! 

Alternatively you could just merge these two subfolders...